flake.nix

```diff
@@ -410,6 +416,11 @@
               postUnpack = "sourceRoot=$sourceRoot/perl";
             });
 
+            passthru.python-bindings = final.callPackage ./python {
+              inherit self system;
+              python = final.python3;
+            };
```

Comment:
It would be nice to have an explicit function as the public interface here. Something like `nix.pythonPackageFor { python = ...; }`.

We could build and test against multiple python versions in CI.